libpod/container_api.go

```diff
@@ -921,7 +921,11 @@ func (c *Container) Stat(ctx context.Context, containerPath string) (*define.Fil
 		if err != nil {
 			return nil, err
 		}
-		defer c.unmount(false)
+		defer func() {
+			if err := c.unmount(false); err != nil {
+				logrus.Errorf("Unmounting container %s: %v", c.ID(), err)
```

Comment:
Should we check if the container is already unmounted?  And only write at Info level in that case.   I am a little worried about a race condition, between --rm and podman container cleanup.